src/Header.tsx

```diff
@@ -33,18 +33,27 @@ function NavItem(props: { href: string; title: string }): JSX.Element {
 }
 
 export default function Header(): JSX.Element {
+  const [activeKey, setActiveKey] = useState<string | null>('/')
```

Comment:
There should be more comprehensive tests. You take the handling over from react-bootstrap and set the default active key to non-existing value. This behavior seems incorrect if the user navigates directly to somewhere else in than the root page. You should tests that correct item is still active in that case.

Reply:
Im not sure i understand. By navigating directly do you mean going directly to https://ebabylon.org/fragmentarium ? instead of https://ebabylon.org ? Should i test that ? It works in praxis.

Reply:
I can write a test for making sure activeKey is set on "manual" redirect. As i said state wouldnt be set on direct navigation via link.

Reply:
> Im not sure i understand. By navigating directly do you mean going directly to https://ebabylon.org/fragmentarium ? instead of https://ebabylon.org ? Should i test that ? It works in praxis.

Yes, this used to work without manual state handling.

Reply:
> So it works as expected when using `null` as default key. When the user navigates directly the `activeKey` is null. When he afterwords decides to click on `Fragmentarium` for example the `activeKey` is `"/fragmnetarium"`. I would suspect thats how bootstrap deals with it. If we want to manually set `activeKey` on "direct navigation" (link for example). We would have to manually get the URL state with react-router and set it. It won't change anything funcitonality wise though (Everything is working currently).

Why we would have to set the active key manually if it would not change anything? Isn't the root cause that the `Nav` does not change active key on link click if a nav link has been clicked before? The current fix is a workaround for one specific link (`Nav` seems only concerned on the actual `Nav.Item`s not whatever may be inside `NavBar`). This could be sufficient now since there are not many cross section links. However, I was able to get two active by following links the glossary:
![image](https://user-images.githubusercontent.com/404469/97587600-da024680-19fb-11eb-86db-ab8b3afe23a1.png)

The real fix would be to make the `Nav` handle route changes correctly (It appears already to be sensitive to the route on the initial render), but that should be done within react-bootstrap. Do you want to open an issue?

Reply:
im not open an issue but i added tests